camera_to_hls_nerves/README.md

```diff
@@ -0,0 +1,132 @@
+# Membrane Demo - Camera Video to HLS on Nerves
+
+This demo demonstrates capturing video from a camera module on Raspberry Pi running Nerves and broadcasting it to a browser via HLS.
+
+## Hardware Prerequisites
+
+To run this demo you'll need a Raspberry Pi and a official Raspberry Pi camera module. Currently the following devices are supported:
+
+* Raspberry Pi 4 Model B
+
+## Software Prerequisites
+
+<details>
+<summary>
+<b>MacOS</b>
+</summary>
+
+To run the demo, you need [Elixir and Erlang installed](https://elixir-lang.org/install.html) on your machine (it's best to use a version manager, like `asdf`). Additionally the following Homebrew command will install all the required packages for nerves to work:
```

Comment:
I'm not a fan of these fancy collapsing blocks, we're removing them from the other demos :P Let's extract what's common to all OSes (elixir and nerves_bootstrap) and convert the rest to normal paragraphs ;)